### DISCUSSION TITLE
Can i play mp3 from spiffs in esp32?

### DISCUSSION THREAD

**Discussion by rickyanwar** [Makuna/DFMiniMp3#62] [Q&A] [answered]
Can i play mp3 in spiffs file, because i dont need micro sd just for 200 kb data

**Comment by Makuna** [comment, marked as answer]
You can not play any audio that is not already present on the media on the device as there is no way to transfer audio data to or from the mp3 device.  

You can get some cheap 2mb micro SD cards if you look around; but they are getting pretty rare now,

**Comment by rickyanwar** [comment]
I mean like this, but using dplayer modul
https://github.com/earlephilhower/ESP8266Audio/tree/master/examples/PlayMIDIFromSPIFFS

**Reply by Makuna** [reply]
I already answered this; the module this library supports DOES NOT HAVE THE ABILITY to send or receive audio from the uC.

**Reply by rickyanwar** [reply]
Hmm thanks you
